nova/manifests/compute.pp

```diff
@@ -131,6 +131,9 @@
   $internal_service_availability_zone = 'internal',
   $heal_instance_info_cache_interval  = '60',
   $pci_passthrough                    = undef,
+  $auth_protocol                      = 'http',
+  $auth_uri                           = 'http://127.0.0.1:5000/v2.0',
+  $identity_uri                       = 'http://127.0.0.1:35357',
```

Comment:
These are the default values. These get overwritten with the values we are passing to this class. For default values, the puppet modules use http and localhost.